app/views/settings/applications/_fields.html.haml

```diff
@@ -7,7 +7,7 @@
 .fields-group
   = f.input :redirect_uri, wrapper: :with_block_label, label: t('activerecord.attributes.doorkeeper/application.redirect_uri'), hint: t('doorkeeper.applications.help.redirect_uri')
 
-  %p.hint= t('doorkeeper.applications.help.native_redirect_uri', native_redirect_uri: Doorkeeper.configuration.native_redirect_uri)
+  %p.hint= t('doorkeeper.applications.help.native_redirect_uri', native_redirect_uri: "<code>#{Doorkeeper.configuration.native_redirect_uri}</code>")
```

Comment:
That's going to get escaped because this string is not marked as html-safe.

Reply:
By Googling, I should use `.html_safe` at the end of string in order for this to work, yes?